### PR TITLE
fix: make test command succeeds locally

### PR DESCRIPTION
**Pull request by mateusoliveira43**
Fix `make test` command so developers can run it locally without false positives.

### CODE REVIEW

File: config/manifests/bases/oadp-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ metadata:
     containerImage: quay.io/konveyor/oadp-operator:latest
     createdAt: "2020-09-08T12:21:00Z"
     description: OADP (OpenShift API for Data Protection) operator sets up and installs
-      Data Protection Applications on the OpenShift platform.
+      Data Protection Applications on the OpenShift platform. Fake edit :)
```
Comment (kaovilai) [inline config/manifests/bases/oadp-operator.clusterserviceversion.yaml:11]:
```suggestion
      Data Protection Applications on the OpenShift platform.
```

Comment (mateusoliveira43) [reply config/manifests/bases/oadp-operator.clusterserviceversion.yaml]: Thanks, was using this to test if it also works with CI ehheeh

**Review by kaovilai — DISMISSED** [review]

**Review by sseago — DISMISSED** [review]

**Review by sseago — APPROVED** [review]

**Review by kaovilai — APPROVED** [review]
/unhold

**Pull request merged**